### DISCUSSION TITLE
[OpenIDC/mod_auth_openidc] Permission Denied (13)

### DISCUSSION THREAD

**Discussion by wellsbj2** [OpenIDC/mod_auth_openidc#1100] [General]
LoadModule auth_openidc_module modules/mod_auth_openidc.so

When hitting port 80 on the server im getting some permission denied (13) errors in the error_log file for Apache,

[Mon Sep 04 13:18:19.676990 2023] [auth_openidc:error] [pid 6816176:tid 515] oidc_cache_mutex_lock: apr_global_mutex_lock() failed: Permission denied (13)
[Mon Sep 04 13:18:19.710943 2023] [auth_openidc:error] [pid 6816176:tid 515] oidc_cache_mutex_unlock: apr_global_mutex_unlock() failed: Permission denied (13)

The apachectl start is run as root and the User Group in httpd.conf is http : www.

Seems ok if I run apache as http on port 8080. Thank you, Brad

**Comment by zandbelt** [comment]
It seems you're trying to run this on AIX but that is only supported under a commercial agreement with licensed binaries built for AIX.

**Reply by wellsbj2** [reply]
Hey, ok thanks.  I have built the module from the source code, no errors came up on the configure, make or make install. Is that not a valid way to build / use this module for AIX?  Thanks..
